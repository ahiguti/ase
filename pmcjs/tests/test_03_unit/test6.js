
var obj = {
  foo: 'val-foo',
  bar: 'val-bar',
  baz: 15
};

var arr = [
  "abc", "def", "ghi"
];


Perl.Use("IterTest");
var s = Perl.Functions["IterTest"].hjoin(obj);
print(s);
s = Perl.Functions["IterTest"].ajoin(arr);
print(s);
