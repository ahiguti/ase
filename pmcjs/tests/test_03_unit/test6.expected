 (foo val-foo) (bar val-bar) (baz 15)
 (0 abc) (1 def) (2 ghi)
